packages/client/src/entities/Forum/topic/ui/TopicListItems/TopicListItems.tsx

```diff
@@ -37,14 +41,15 @@ export const TopicListItems = () => {
         itemLayout="vertical"
         dataSource={topics}
         renderItem={topic => (
-          <List.Item>
+          <List.Item key={topic.id}>
             <Card
               className={styles.cardHover}
               title={topic.title}
-              onClick={() => handleSelectTopic(topic.id)}>
+              onClick={() => handleSelectTopic(topic.id)}
+              hoverable>
               <p>{topic.content.substring(0, 120)}...</p>
               <div className={styles['container-card-extra']}>
-                <Text type="secondary">@{topic.author}</Text>
+                <Text type="secondary">@{topic.userId}</Text>
```

Comment:
```suggestion
                <Text type="secondary">@{topic.userId}</Text>
```

Не стоит явно выводить id ползователя, лучше так же как в комментариях сделать никнейм или почту